students/csdotson/lesson03/mailroom.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# Lesson 3 - Mailroom... part 1
+
+# Initial list of donors/donations
+donations = [ ["Bill Gates", 10.50], ["Bill Gates", 123.45], ["Bill Gates",    1111.11], ["Jeff Bezos", 7.65], ["Jeff Bezos", 1000], ["Paul Allen", 145.90], ["John Nordstrom", 45.67], ["John Nordstrom", 6519.65], ["Mark Zuck", 789.12] ]
+
+
+### Function Definitions ###
+def prompt(which = 0):
+    # Original prompt for main menu selections
+    if which == 0:
+        response = input("Welcome Chris! What would you like to do?\n(S)end a Thank You / (C)reate a Report / (Q)uit\n--> ")
+    else:
+        response = input("What would you like to do next?\n(S)end a Thank You / (C)reate a Report / (Q)uit\n--> ")
+
+    while True:
+        if response == "S":
+            send_thank_you()
+            break
+        elif response == "C":
+            print_report()
+            break
+        else:
+            print("Goodbye Sir!")
+            break
+
+def send_thank_you():
+    # Add donor/donations and trigger thank you email
+    response = input("Please enter full name of person you'd like to thank:\nBy the way, you can also type 'list' to get a list of donors or 'P' to return to the main prompt\n--> ")
+
+    if response == "list":
+        print()
+        list_donors()
+        send_thank_you()
+    elif response == "P":
+        prompt(1)
```

Comment:
magic numbers 😉 (https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

you could define constants for cases of `which` var -> `DFLT_PROMPT = 0, NEXT_PROMPT = 1`